src/sst/elements/cramSim/tests/testsuite_default_cramSim.py

```diff
@@ -112,6 +132,9 @@ def cramSim_test_template(self, testcase):
 #####
 
     def _setupcramSimTestFiles(self):
+        if not self.__class__.tmp_file:
+            # if tmp_file is None setUpClass() never ran
+            self.skipTest(f"{self} cannot be run concurrently")
```

Comment:
Is setUpClass skipped when running concurrently? Or may be run multiple times?

Reply:
It's skipped... I believe it has something to do with testtools but I haven't had a chance to dig deep enough yet.

Reply:
OK. We may want to revisit how we do one time setup in general since it'd be nice to support concurrent tests again. Looks fine for now though.